OVERVIEW.md

```diff
@@ -221,12 +221,19 @@ Extends defined rules with preset rules.
 
 Type: `String`
 
-Values: `"airbnb"`, `"crockford"`, `"google"`, `"jquery"`, `"mdcs"`, `"node-style-guide"`, `"wikimedia"`, `"wordpress"`, `"yandex"`
+Values: You can choose one of the default presets: `"airbnb"`, `"crockford"`, `"google"`, `"jquery"`, `"mdcs"`, `"node-style-guide"`, `"wikimedia"`, `"wordpress"`, `"yandex"`.
+
+Or you can load from it local path or as node module
 
 #### Example
 
 ```js
 "preset": "jquery"
+"preset": "./path-to-your-preset"
+
+// If your preset called "jscs-your-preset-node_modules-path" 
+// You can either define full name or omit "jscs-" prefix -
```

Comment:
Prefixes usually added so they wouldn't be confused with other packages since it name without prefix might be misleading and have possibility of the name conflict - https://twitter.com/ironshay/status/370525864523743232

ESlint recent change of the same nature and popularity of https://github.com/sindresorhus/load-grunt-tasks packages somewhat proves it, plus it seems popular for the jscs too - https://www.npmjs.com/search?q=jscs

So we know ppl use those prefixes a lot, so we might take advantage of it...

And you know, it is nicer to write `preset: my-company`, then `preset: jscs-my-company`.